job_templates/sag_nemo/app_1/config_fed_client.conf

```diff
@@ -65,14 +59,19 @@
 
   components =  [
     {
-      # This "launcher" component
+      # component id is "launcher"
       id = "launcher"
 
-      # the class path of the component
+      # the class path of this component
       path = "nvflare.app_common.launchers.subprocess_launcher.SubprocessLauncher"
 
-      # the launcher will invoke the script
-      args.script = "python3 -m torch.distributed.run {ddp_config} custom/{app_script}  {app_config} "
```

Comment:
Shouldn't this remain a distributed run? We need to support multi-gpu training for nemo.

Reply:
yes, it should, I copy and paste and accidentally change this, thanks for catching this